pkg/golinters/gomnd.go

```diff
@@ -9,10 +9,23 @@ import (
 )
 
 func NewGoMND(settings *config.GoMndSettings) *goanalysis.Linter {
+	// The constant is only used to force the analyzer name to use the same name as the linter.
+	// This is required to avoid displaying the analyzer name inside the issue text.
```

Comment:
Not sure I follow the rationale here. Is it OK to change the output from `err113` because there are no settings but for linters with settings it would be confusing to configure it as `gomnd` but only see `(mnd)` in the output?

Since we use `analyzer.Name` for most linters, what happens if someone changes the name of their linter?

Reply:
> Is it OK to change the output from err113 because there are no settings but for linters with settings it would be confusing to configure it as gomnd but only see (mnd) in the output?

The problem is not the output but the name of the linter inside `enable`/`disable` section and the name of the linter inside the configuration.

> Since we use analyzer.Name for most linters, what happens if someone changes the name of their linter?

First, you have to know that, currently, we have linters that are multiple analyzers (ex: govet)

If someone changes the name of an analyzer, we can inside golangci-lint, override it to keep the compatibility.

Reply:
Ah, right! Thanks for the explanation!